### ISSUE TITLE
mkcert -install not working correctly for Google Chrome on Ubuntu 18.04 (no trusted session)

### ISSUE THREAD

**Issue by bfuzze** [drud/ddev#2436]
**Describe the bug**
I followed the mkcert instructions here: https://ddev.readthedocs.io/en/stable/#linux-mkcert-install-additional-instructions
I was able to create and install a certificate, but the site SSL validation was still failing after numerous restarts/attempts.

Eventually (and by chance), I discovered that the .pem added to the Chrome trust store was incorrect.  Under /etc/ssl/certs/ I found mkcert_development_CA_####.  I copied this to the root of my project, imported it into the Chrome authorities (had to remove the original first) and the site certificate started working.

**To Reproduce**
I followed the mkcert instructions here: https://ddev.readthedocs.io/en/stable/#linux-mkcert-install-additional-instructions

**Expected behavior**
The site SSL certificate should validate after following the documented steps.

**Screenshots**
![image](https://user-images.githubusercontent.com/1250453/89143463-a7d66b00-d518-11ea-8748-8f3625316450.png)

**Version and configuration information (please complete the following information):**
Ubuntu 18.04
ddev version v1.14.2
Docker version 19.03.6, build 369ce74a3c

**Additional context**
Drupal 8 project

**Comment by rfay** [comment]
The correct way to remove would be to `mkcert -uninstall`. You could then completely destroy existing CAs by doing `rm -r "$(mkcert -CAROOT)". You could then `mkcert -install` again from scratch.

People use this with Chromium at least all the time, I don't know if that's what you were using. But as the doc says, maybe you didn't have certutil?

**Comment by rfay** [comment]
I don't seem to have enough room on my system to get out my 18.04 VM right now, everybody seems to have moved on to 20.04, so I've had that one handy. But perhaps you can do a `mkcert --version` to make sure you have the current version, and if you don't, update it. If you're using homebrew, you can update it with ddev. Of course we know from the other issue you can also update ddev with homebrew if that's how you're installing. Binaries are at https://github.com/FiloSottile/mkcert/releases, if that's how you do it. v1.4.1 is the current release.

**Comment by rfay** [comment]
I finally got Ubuntu 18.04 update and copied back so I could use it. 

I installed Google Chrome by downloading the .deb file from https://www.google.com/chrome/?brand=CHBD&gclid=EAIaIQobChMItNTEuZb_6gIVkIbACh05Ag3SEAAYASAAEgJ9x_D_BwE&gclsrc=aw.ds and `sudo apt- install ./google*.deb`

`mkcert -install` did complain about the new location and suggest `mkcert -uninstall`. So I did `mkcert -uninstall` and then `mkcert -install`, then `ddev poweroff` and `ddev start` and it worked fine.
